pyplanet/apps/contrib/jukebox/views.py

```diff
@@ -105,6 +105,13 @@ async def get_data(self):
 			map_dict['local_record_diff_direction'] = None
 			map_dict['karma'] = None
 
+			# Use custom field for author to allow for searching on both login and nickname (depending on what's shown).
+			map_dict['author'] = (
+				map_dict['author_nickname']
+				if 'author_nickname' in map_dict and map_dict['author_nickname'] and len(map_dict['author_nickname'])
```

Comment:
The len() you can remove, this is checked by the earlier statement.

Reply:
Done.